packages/card/CtaCard.jsx

```diff
@@ -0,0 +1,109 @@
+ * @param {string} externalUrl Manually entered external link
+ * @param {boolean} isNewWindow Whether the link should open in a new window
+ * @return {JSX.element}
+ */
+
+export default function CtaCard({
+  title,
+  eyebrow,
+  description,
+  internalUrl,
+  externalUrl,
+  isNewWindow,
+}) {
+  const hasLink = !!internalUrl || !!externalUrl;
+  // Check extenalUrl field to see if it is actually external
+  // Do not use external arrow and rel attributes if link has news.stanford.edu
+  const isRealExternalLink =
+    !!externalUrl && !externalUrl?.includes("news.stanford.edu");
+
+  return (
+    <article
+      className={cnb(
+        "su-group su-relative su-w-full su-flex su-flex-col su-break-words su-bg-white su-rounded-[8px] su-rs-pt-5 su-rs-px-4 su-max-w-900 su-mx-auto su-transition-shadow su-border su-border-black-30/30 su-shadow",
+        internalUrl || externalUrl
+          ? "hover:su-shadow-md focus-within:su-shadow-md"
+          : "",
+        hasLink ? "su-rs-pb-4" : "su-rs-pb-5"
+      )}
+    >
+      {eyebrow && (
+        <span
+          aria-hidden
+          className="su-type-1 su-text-black-60 su-font-semibold su-rs-mb-1"
+        >
+          {eyebrow}
+        </span>
+      )}
+      <h2 className="su-type-5 md:su-type-4 2xl:su-type-3 su-mb-0 su-font-sans su-text-black">
+        {hasLink ? (
+          // eslint-disable-next-line react/jsx-no-target-blank
+          <a
+            href={internalUrl || externalUrl}
+            target={isNewWindow ? "_blank" : undefined}
+            rel={isRealExternalLink ? "noopener nofollow" : undefined}
+            className="group-hocus-within:su-underline su-stretched-link su-text-black group-hocus-within:su-text-digital-red"
+          >
+            {eyebrow && <span className="su-sr-only">{`${eyebrow}:`}</span>}
+            {title}
+          </a>
+        ) : (
+          <>
+            {eyebrow && <span className="su-sr-only">{`${eyebrow}:`}</span>}
+            {title}
+          </>
+        )}
+      </h2>
+      {!!description && (
+        <div className="su-grow">
+          <XssSafeContent
+            data-test="cta-card-content"
+            className="su-text-black su-big-paragraph su-rs-mt-4 *:su-leading-snug *:last:su-mb-0"
+            content={description}
+          />
+        </div>
+      )}
+      {hasLink && (
+        <div className="su-mt-auto">
+          <div
+            className={cnb(
+              "su-flex su-rs-mt-1 su-transition-transform su-items-center su-justify-center su-size-50 md:su-size-70 su-rounded-full su-bg-gradient-to-r su-from-digital-red-light su-to-cardinal-red-dark su-ml-auto",
+              isRealExternalLink
+                ? "group-hocus-within:su-translate-x-02em group-hocus-within:su--translate-y-02em"
+                : "group-hocus-within:su-translate-x-03em"
+            )}
+          >
+            {isRealExternalLink ? (
+              <ExternalArrowUnstyled
+                title="link is external"
```

Comment:
If the link is external, I added a title for the SVG "link is external". If the link is internal, I aria-hidden it since the chevron icon is decorative only.